corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -291,6 +334,21 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                 },
             };
         },
+
+        onAttach: function () {
+            const self = this;
+            // if (self.isMultiSelect && self.smallScreenEnabled) {
```

Comment:
Assume this should be uncommented; we're limiting to only multi-select case tiles, right?